sklearn/utils/estimator_checks.py

```diff
@@ -186,6 +189,13 @@ def _yield_transformer_checks(name, Transformer):
         # basic tests
         yield check_transformer_general
         yield check_transformers_unfitted
+    # Dependent on external solvers and hence accessing the iter
+    # param is non-trivial.
+    external_solver = ['Isomap', 'KernelPCA', 'LocallyLinearEmbedding',
```

Comment:
Yes, they were skipped before. I agree that finding another solution sounds like a great idea for the future.